README.md

```diff
@@ -181,18 +181,21 @@ Now you'll need to do steps (2) everytime you make a changes and want to build t
       pip install selenium
    ```
 
-4. (Optional) Configure: open `tests/runtests.py` and configure variables as per your requirement, options are:
+4. (Optional) Configure: open `tests/config.json` and configure variables as per your requirement, options are:
 
    ```yaml
       headless: Run selenium chrome driver in headless mode
       app_url: URL to reach the admin dashboard
       username: username for logging in admin dashboard
       password: password for logging in admin dashboard
+      services_max_retries: Maximum number of retries to check if services are running
+      services_retry_wait: Delay time (in seconds) to each retries for checking if services are running
    ```
 
 5. Run tests:
 
    ```bash
+      python test_services.py
```

Comment:
Instead of 2 different files, I would prefer that there is one file running both the tests! :smile:
Like `python runtests`